radiacode-examples/basic.py

```diff
@@ -1,20 +1,40 @@
 import argparse
-import time
+import time, platform
 
 from radiacode import RadiaCode
-
+from radiacode.transports.usb import DeviceNotFound as DeviceNotFoundUSB
+from radiacode.transports.bluetooth import DeviceNotFound as DeviceNotFoundBT
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--bluetooth-mac', type=str, required=False, help='bluetooth MAC address of radiascan device')
+
+    if platform.system() != 'Darwin':
+        parser.add_argument('--bluetooth-mac', type=str, required=False, help='bluetooth MAC address of radiascan device (e.g. 00:11:22:33:44:55)')
+
+    parser.add_argument('--serial', type=str, required=False,
+                        help='serial number of radiascan device (e.g. "RC-10x-xxxxxx"). Useful in case of multiple devices.')
+
     args = parser.parse_args()
 
-    if args.bluetooth_mac:
-        print('will use Bluetooth connection')
-        rc = RadiaCode(bluetooth_mac=args.bluetooth_mac)
+    if hasattr(args, 'bluetooth_mac') and getattr(args, 'bluetooth_mac', None):
```

Comment:
`and getattr(args, 'bluetooth_mac', None):` can be simplified to `and args.bluetooth_mac:`

Reply:
Change applied and pushed in the branch